### PR TITLE
Update the readme, hacking, and brandstores

### PR DESCRIPTION
**Pull request by pmahnke**
# Done

- Updated readme.md to be more new user friendly
- Created a hacking.md for how to develop on snapcraft.io
- Created a brandstore.md for how to work on brand variations of snapcraft.io

# QA

- View the file in this branch (e.g. https://github.com/canonical-web-and-design/snapcraft.io/blob/d8415e5ffb2f515ed77be086e87c796c2867fa91/README.md)
- Make sure the content it good, the links work, you agree with the work

# Note

- I likely need a better image @kwm14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,51 @@
-# snapcraft.io
+# ![snapcraft.io](https://assets.ubuntu.com/v1/f46fbbb9-snapcraft-logo--web.png "Snapcraft").io codebase
```
Comment (bartaz) [inline README.md]: Snapcraft logo looks a bit weird mixed with GH heading styles. Let's keep them separate.

```suggestion
# ![snapcraft.io](https://assets.ubuntu.com/v1/f46fbbb9-snapcraft-logo--web.png "Snapcraft") snapcraft.io codebase
```

File: README.md [hunk 2]
```diff
@@ -1,97 +1,51 @@
...
-
-Create or update you `.env.local` file:
-
-```
-SENTRY_DSN=<DSN_FROM_ABOVE>
-```
+If you are interested in Snaps, Snapping and Snapcraft, there is an active [discourse forum](https://forum.snapcraft.io/) that we encourage developers to join.
 
-The application will be reporting errors to your `sentry.io` project from now on.
+The site is largely maintained by the [Web and Design team](https://ubuntu.com/blog/topics/design) at [Canonical](https://www.canonical.com). It is a simple, stateless website project based on [Flask](https://flask.palletsprojects.com/en/1.1.x/) and hosted on a [Charmed Kubernetes](https://ubuntu.com/kubernetes) cluster.
```
Comment (bartaz) [inline README.md]: "simple" 🤣

Comment (Lukewh) [inline README.md]:
```suggestion
The site is largely maintained by the [Web and Design team](https://ubuntu.com/blog/topics/design) at [Canonical](https://www.canonical.com). It is a stateless website project based on [Flask](https://flask.palletsprojects.com/en/1.1.x/) and hosted on a [Charmed Kubernetes](https://ubuntu.com/kubernetes) cluster.
```

Comment (pmahnke) [reply README.md]: I might have borrowed that line from ubuntu.com

**Comment by bartaz** [comment]
Thanks for the updates @pmahnke, the logo in heading still looks weird (as it's misaligned with text):

Screenshot 2019-10-08 at 08 02 03

Could we just maybe move it out completely above the title?

Screenshot 2019-10-08 at 08 03 25

**Comment by pmahnke** [comment]
@bartaz how about this

image

**Comment by bartaz** [comment]
@pmahnke oh, that's a good idea! LGTM.

**Pull request merged**